.github/workflows/CI.yml

```diff
@@ -218,37 +237,51 @@ jobs:
             mingw-w64-x86_64-opus 
             mingw-w64-x86_64-x265
             mingw-w64-x86_64-boost 
+            mingw-w64-x86_64-nsis
             git 
             yasm 
             nasm 
             diffutils 
             make
       - name: Build Windows
         shell: msys2 {0}
-        run: |
-          mkdir sunshine-windows-build && cd sunshine-windows-build
+        run: |   
+          mkdir sunshine-windows-build
+          cd sunshine-windows-build
           cmake -DCMAKE_BUILD_TYPE=Release -DSUNSHINE_ASSETS_DIR=assets -G "MinGW Makefiles" ..
           mingw32-make -j2
-      - name: Package Windows
+      - name: Create packages
+        shell: msys2 {0}
         run: |
           cd sunshine-windows-build
-          del ..\assets\apps_linux.json
-          7z a sunshine-windows.zip ..\assets
-          7z a sunshine-windows.zip sunshine.exe
-          7z a sunshine-windows.zip tools\dxgi-info.exe
-          7z a sunshine-windows.zip tools\audio-info.exe
-          7z a sunshine-windows.zip tools\sunshinesvc.exe
-          7z a sunshine-windows.zip ..\tools\install-service.bat
-          7z a sunshine-windows.zip ..\tools\uninstall-service.bat
-          cd ..
-          mkdir artifacts
-          move "sunshine-windows-build\sunshine-windows.zip" "artifacts"
-      - name: Upload Artifacts
+          cpack
+
+          mkdir -p artifacts/standalone
+          mkdir -p artifacts/web
+          mkdir -p artifacts/shaders/directx
+          
+          # Installers
+          mv Sunshine__.exe artifacts/sunshine-windows-installer.exe
+          
+          # Standalone
+          mv ../assets/web artifacts/web
+          mv tools artifacts/tools
+          mv ../assets/shaders/directx artifacts/shaders/directx
+          mv ../assets/apps_windows.json artifacts/apps_windows.json
+          mv ../assets/sunshine.conf artifacts/sunshine.conf
+          mv sunshine.exe artifacts/sunshine.exe
+      - name: Upload Sunshine Installer
+        if: ${{ github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: sunshine-windows-installer
+          path: artifacts/sunshine-windows-installer.exe
```

Comment:
This won't be necessary because just using `path: artifacts` will upload all the files that exist in that folder (this will be the same behavior for the create release action)

Reply:
The problem with this is that Github will make one `.zip` file that contains both the `.exe` installer and the `.zip` with the sunshine standalone package resulting in a 20MB zip to be donwloaded that basically contains everything twice just packaged in two different formats (see: https://github.com/ABeltramo/Sunshine/actions/runs/2249492914).

I think it's better to split them in two separate 10MB artifacts so that users can download only the one needed.

Reply:
It only does it for artifacts though, not releases. Users are rarely downloading artifacts.

Just trying to keep the CI more simple as it's already overly complex.